api/src/main/java/org/pmiops/workbench/db/model/DbCdrVersion.java

```diff
@@ -38,7 +35,6 @@
   private Boolean hasCopeSurveyData;
 
   @Id
-  @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
This was causing a bug in UpdateCdrConfig: it would sometimes create new `cdr_version` entities with new generated IDs instead of updating existing rows in place with the IDs we specify.

Fortunately I was able to confirm that all rows in all environments have the correct IDs - except for my local, temporarily.